packages/react-components-pro/package.json

```diff
@@ -39,18 +39,8 @@
   "author": "Vaadin Ltd.",
   "license": "SEE LICENSE IN LICENSE",
   "peerDependencies": {
-    "@types/react": "^18.2.37",
```

Comment:
What about `@types/react`? Should we still keep them?

Reply:
You tell me, what is the purpose of them? Haven't seen them in many other react projects

Reply:
Let's do a minimal change then if we don't really know